### PR TITLE
Fixed grammar in proposed Event Manager PSR

### PR DESCRIPTION
**Pull request by SidRoberts**

### CODE REVIEW

**Comment by michaelcullum** [comment]
@php-fig/psr-14

File: proposed/event-manager.md [hunk 1]
```diff
@@ -147,19 +148,19 @@ interface EventManagerInterface
...
-     * @return bool true on success false on failure
+     * @return bool true on success; false on failure
      */
     public function attach($event, $callback, $priority = 0);
 
     /**
      * Detaches a listener from an event
      *
-     * @param string $event the event to attach too
+     * @param string $event the event to attach to
```
Comment (harikt) [inline proposed/event-manager.md]: Isn't this should be detach or remove? or is it overwriting the event itself. Slow connection unable to send a PR.

Comment (SidRoberts) [inline proposed/event-manager.md]: You're right. :) I've fixed it in 6b10147.

**Comment by Jean85** [comment]
The PSR is now back in active development, there are now conflicts. Closing this.

**Pull request closed without merging**